### DISCUSSION TITLE
2024-08-14 수요일

### DISCUSSION THREAD

**Discussion by eunohhh** [nbc-final-a10/TripBuddy#265] [기록보드]
2024-07-26 금요일 기록보드입니다.

## 여행가고십조(A10) 일일 공통 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 아침 회의 | |
| 10:00 ~ 11:00 |  스파르타 채용설명회 (zoom | |
| 11:00 ~ 13:00 | 집중 개발 time | |
| 13:00 ~ 14:00 | 점심 식사 | |
| 14:00 ~ 18:00 | 집중 개발 time | |
| 18:00 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | 저녁 회의 | |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | |

```
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 내용

### 트러블 슈팅

- 내용

### 내일 계획

- 내용
```

```
// 이미지 테이블
<table>
  <tr>
    <th>이미지 1번 제목</th>
    <th>이미지 2번 제목</th>
  </tr>
  <tr>
    <td><img src="이미지 주소" width="300" height="300"/></td>
    <td><img src="이미지 주소" width="300" height="300"/></td>
  </tr>
</table>
```

**Comment by eunohhh** [comment]
## 개인 기록보드 서식 (오은)

### 처리 할 사항

- [ ] : UI 정돈 전체적으로
- [ ] : 탑배너 프리페치쿼리
- [ ] : 스토리 관련 로직 전체 검토
- [x] : 얼러트 모달 수정
- [x] : 여정 리스트 인기순 / 최신순 필터링
- [x] : 여정 리스트 모바일에서 필터 버튼 만들기
- [x] : 여정 카드 hot 로직
- [x] : 여정 카드 국내/국외 로직
- [x] : 스토리 카드에서 하트 삭제
- [x] : 글쓰기페이지 UI 안짤리게
- [x] : 상대플필보기에서 여정보기 시에도 화면 지진현상
- [x] : ai 이미지
- [x] : 글쓰기 페이지 레이아웃 검토
- [ ] : 오늘날짜를 여정 시작일로 할 경우 뜨는 에러 처리
- [x] : 컨디셔널 라우트 - 병준님이 처리하신듯
- [x] : 온보딩으로 보내기처리 - 사인업일때는 쉬움 / 프로바이더일때 처리요망... 완료

### 처리 된 사항
- [x] : 여정 카드에 플필사진 4개까지 보이게끔
- [x] : 여정 찜하기 
- [x] : 여정 '나기기'표시
- [x] : 글수정 페이지 만들기 모바일
- [x] : 글수정 페이지 만들기 데탑
- [x] : 비번찾기 뒤로가기 동작 수정
- [x] : 스토리 글쓰기 시 옵션 조금 상세하게 
- [x] : 튜토리얼 데스크탑에서 헤더 숨기기 
- [x] : pwa 연구 - 하단 참조(ios에서는 불가능)
- [x] : 스토리 삭제 
- [x] : 스토리 좋아요 
- [x] : 스토리 좋아요 옵티미스틱 
- [x] : 모바일에서 스토리 인풋문제 
- [x] : 홈화면 카드에서도 스토리 좋아요 노출하기
- [x] : 네이버 검수요청 
- [x] : 미들웨어 설정
- [x] : 여정 데스크탑 페이지네이션 
- [x] : 여정 관련 UI 수정작업
- [x] : 여정 모바일 무한스크롤
- [x] : 스토리 데탑/모바일 위치 맞추기
- [x] : 데스크탑 홈에서 마우스 스크롤 동작 수정 - 세로스크롤 가능하게하고, 화살표로 목록 이동하고, 좌우스크롤은 옵션
- [x] : error.js not-found.js 등 없는 기능들 
- [x] : 홈배너 로직 수정 - 탠스택쿼리
- [x] : 메인 스테일타임 점검 - 5분으로 설정
- [x] : 하단바 svgr 및 fixed
- [x] : 프로필 에디트 라우트 이동
- [x] : UT 피드백 처리 

### 보류 사항
- [ ] : 회원가입 디바운싱 ??
- [ ] : 여정 임시 저장 ??

**Comment by Captain-Kim** [comment]
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 팔로우 버튼 버디 프로필과 여정 상세에서 사용 가능하도록 유연하게 수정

### 트러블 슈팅

- 팔로우 버튼을 여정 상세와 버디 프로필에서 사용 가능하게 수정하는 과정에서 window 객체에서 url로 판별하는 로직을 사용하고 있는데 (이것 뿐만 아니라 재사용하는 컴포넌트들을 본인은 그런 식으로 작성했었음)
  - 문제는 window 객체는 DOM 요소가 생성되고 나서 불러올 수 있기 때문에 fetching이 너무 느려짐.
  - 더 좋은 방법을 찾아서 리팩토링 해야 할 것 같음.
  - => useRouter 훅으로 가져오는 방법을 찾아봐야겠음.

### 내일 계획

- 내용
